docs/src/ComboBox.doc.js

```diff
@@ -0,0 +1,852 @@
+        errorMessage={errorMessage}
+        id="displayLabel"
+        label="Choose a category to display top search trends"
+        labelDisplay="hidden"
+        noResultText="No results for your selection"
+        onBlur={handleOnBlur}
+        onChange={resetErrorMessage}
+        onClear={resetErrorMessage}
+        options={options}
+        placeholder="Select category"
+      />
+    </Flex>
+  );
+}
+`}
+      />
+      <MainSection.Subsection
+        title="Keyboard interaction"
+        description={`
+    * Hitting \`Enter\` or \`Space\` key on the ComboBox's trigger opens the options list
+    * Once an item is selected, hitting \`Enter\` or \`Space\` on the clear button clears the selection and returns focus to the input textfield
+    * \`Escape\` key closes the options list, while moving focus back on the ComboBox's trigger
+    * Arrow keys are used to navigate items within the options list
+    * \`Enter\` key selects an item within the options list
+    * \`Tab\` or \` Shift + Tab\` close the options list and move focus accordingly
+  `}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection
+    name="Localization"
+    description={`Be sure to localize the \`helperText\`, \`errorMessage\`, \`noResultText\`, \`label\`, \`placeholder\`,  and \`accessibilityClearButtonLabel\` props. \`options\` and \`value\` should be localized for those cases that can be translated. Note that localization can lengthen text by 20 to 30 percent.`}
+  >
+    <MainSection.Card
+      cardSize="lg"
+      defaultCode={`
+function ComboBoxExample(props) {
+  const PRONOUNS = [
+    'ell@ / l@ / -@',
+    'ella / la / le / -a',
+    'elle / le / -e',
+    'ellx / lx / -x',
+    'él / lo / le / -o',
+  ];
+
+  const options = PRONOUNS.map((pronoun, index) => ({ label: pronoun, value: 'value'+index }));
+
+  const [errorMessage, setErrorMessage] = React.useState();
+
+  const handleOnBlur = ({ value }) => {
+    if (value !== "" && !PRONOUNS.includes(value)) setErrorMessage('Por favor, selecciona una opción válida');
+  };
+
+  const resetErrorMessage = () => (errorMessage ? setErrorMessage() : () => {});
+
+  return (
+    <Box width={400}>
+      <ComboBox
+        accessibilityClearButtonLabel="Remueve la lista de pronombres seleccionados"
+        errorMessage={errorMessage}
+        helperText="Elige hasta 2 grupos de pronombres para que aparezcan en tu perfil y otras personas sepan cómo referirse a ti. Puedes editarlos o eliminarlos en cualquier momento."
+        id="localization"
+        label="Pronombres"
+        noResultText="No se encontró ninguna coincidencia"
+        onBlur={handleOnBlur}
+        onChange={resetErrorMessage}
+        onClear={resetErrorMessage}
+        options={options}
+        placeholder="Añade tus pronombres"
+      />
+    </Box>
+
+  );
+}
+`}
+    />
+  </MainSection>,
+);
+
+card(
+  <MainSection name="Variants">
```

Comment:
Just realized we should probably show an "Error" variant example as well

Reply:
I just did a static error variant. Some of them, uncontrolled,  show the error when Textfield input value is invalid on `onBlur`

Reply:
DONE